modules/e2ee/E2EEContext.js

```diff
@@ -37,14 +37,22 @@ export default class E2EEcontext {
             baseUrl = `${ljm.src.substring(0, idx)}/`;
         }
 
-        // Initialize the E2EE worker. In order to avoid CORS issues, start the worker and have it
-        // synchronously load the JS.
         const workerUrl = `${baseUrl}lib-jitsi-meet.e2ee-worker.js`;
-        const workerBlob
-            = new Blob([ `importScripts("${workerUrl}");` ], { type: 'application/javascript' });
-        const blobUrl = window.URL.createObjectURL(workerBlob);
 
-        this._worker = new Worker(blobUrl, { name: 'E2EE Worker' });
+        if (baseUrl && baseUrl.length > 0) {
+            // Initialize the E2EE worker. In order to avoid CORS issues, start the worker and have it
+            // synchronously load the JS.
+            const workerBlob
+                = new Blob([ `importScripts("${workerUrl}");` ], { type: 'application/javascript' });
+            const blobUrl = window.URL.createObjectURL(workerBlob);
+
+            this._worker = new Worker(blobUrl, { name: 'E2EE Worker' });
```

Comment:
Let's turn workerBlob into a let and reassign it in this case. Then we can have a single worker construction.

Reply:
Done!